<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1613</width>
    <height>568</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1613</width>
    <height>568</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolTipDuration">
   <number>0</number>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(209, 209, 209);
background-color: rgb(60, 113, 156);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="ShowMsg">
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>None  Input</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="Stop_D">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>66</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>  Stop  </string>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="autoRepeatInterval">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Stop_S">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>66</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>  Stop  </string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="Detect">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>-&gt;     Detect     &lt;-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="DetMsg">
      <property name="minimumSize">
       <size>
        <width>330</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>None Output</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="6">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelOrg">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>请选择输入源</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelDet">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>请选择检测器</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Imshow">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <stylestrategy>PreferAntialias</stylestrategy>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>-&gt;     ImShow     &lt;-</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1613</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(125, 203, 255);
color: rgb(24, 64, 143);</string>
   </property>
   <widget class="QMenu" name="menufile_F">
    <property name="geometry">
     <rect>
      <x>286</x>
      <y>204</y>
      <width>129</width>
      <height>213</height>
     </rect>
    </property>
    <property name="title">
     <string>Model(M)</string>
    </property>
    <widget class="QMenu" name="menutensorRT">
     <property name="title">
      <string>TensorRT</string>
     </property>
     <addaction name="actionINT8"/>
     <addaction name="actionFP16"/>
     <addaction name="actionFP32"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionDarknet"/>
    <addaction name="separator"/>
    <addaction name="menutensorRT"/>
    <addaction name="separator"/>
    <addaction name="actionTorchSpt"/>
    <addaction name="separator"/>
    <addaction name="actionOnnx"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuhelp_2">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionCopyRight"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuSave">
    <property name="title">
     <string>Save</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action2Json"/>
    <addaction name="separator"/>
    <addaction name="action2TxT"/>
    <addaction name="separator"/>
    <addaction name="action2Img"/>
    <addaction name="separator"/>
    <addaction name="action2NoSave"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuIO_S">
    <property name="title">
     <string>IO(S)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionImage"/>
    <addaction name="separator"/>
    <addaction name="actionVideo"/>
    <addaction name="separator"/>
    <addaction name="actionCamera"/>
    <addaction name="separator"/>
    <addaction name="actionFolder"/>
    <addaction name="separator"/>
    <addaction name="actionTxT"/>
    <addaction name="separator"/>
    <addaction name="actionExitQ"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuIO_S"/>
   <addaction name="menufile_F"/>
   <addaction name="menuSave"/>
   <addaction name="menuhelp_2"/>
  </widget>
  <action name="actionExitQ">
   <property name="text">
    <string>Exit(Q)</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Image</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionVideo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Video</string>
   </property>
  </action>
  <action name="actionTxT">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TxT</string>
   </property>
  </action>
  <action name="actionFolder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Folder</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
  </action>
  <action name="actionINT8">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>INT-8</string>
   </property>
  </action>
  <action name="actionFP16">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FP-16</string>
   </property>
  </action>
  <action name="actionFP32">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>FP-32</string>
   </property>
  </action>
  <action name="actionDarknet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DarkNet</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionTorchSpt">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TorchScript</string>
   </property>
  </action>
  <action name="actionOnnx">
   <property name="text">
    <string>Onnx</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionCopyRight">
   <property name="text">
    <string>Copy@Right</string>
   </property>
  </action>
  <action name="action2Json">
   <property name="text">
    <string>Json</string>
   </property>
  </action>
  <action name="action2TxT">
   <property name="text">
    <string>TxT</string>
   </property>
  </action>
  <action name="action2Img">
   <property name="text">
    <string>ImG</string>
   </property>
  </action>
  <action name="action2NoSave">
   <property name="text">
    <string>DisAble</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="names.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExitQ</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
